xcode/Subconscious/Shared/Models/Slug.swift

```diff
@@ -69,16 +76,37 @@ public struct Slug:
     
     /// Excludes "internal" slugs like `Slug.profile`
     public var isHidden: Bool {
-        verbatim.hasPrefix("_")
+        verbatim.hasPrefix(Self.hiddenPrefix)
+    }
+    
+    public init?(hidden: String) {
```

Comment:
Noted during our last chat, but I feel we should reverse this relationship in order to uphold the spirit of the `LosslessStringConvertible` protocol (allow hidden is the default constructor, and have an additional "safe" constructor for only visible/user-visible slugs)

Reply:
Updated this, I scanned over tall the call sites for the `Slug.init()` constuctor but based on playtesting I think everything is fine as-is (with the updating `Slug.init(formatting:)` constructor)